meinberlin/templates/a4modules/module_detail.html

```diff
@@ -91,7 +91,6 @@ <h1 class="project-header__title">{{ module.name }}</h1>
         {% with phase_count=module.phases.count %}
         <div class="l-center-6">
             {% for phase in module.phases %}
-                {% if phase.is_over or phase == module.active_phase %}
```

Comment:
everything we do in the module detail, we have to also do in the project detail in many cases. https://github.com/liqd/a4-meinberlin/blob/b079c853aa08e033fd882b718159966391262161/meinberlin/apps/projects/templates/meinberlin_projects/project_detail.html#L214 
I guess putting that into includes would be a good idea... Will add an issue!

Reply:
Oh yes, I forgot that the phases are shown there for single module!